pkg/kubectl/cmd/get/get.go

```diff
@@ -299,8 +333,8 @@ func (options *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []str
 		return err
 	}
 
-	if printer.IsGeneric() {
-		return options.printGeneric(printer, r)
+	if options.PrintGeneric {
+		return options.printGeneric(r)
```

Comment:
The `printGeneric` method is not about just printing, let's rename it to `getGeneric` or alike. The word `print` in it is confusing with the printers. wdyt?